packages/rocketchat-channel-settings/client/views/channelSettings.js

```diff
@@ -54,7 +54,7 @@ Template.channelSettingsEditing.events({
 });
 
 Template.channelSettingsEditing.onCreated(function() {
-	const room = this.room = ChatRoom.findOne(this.data && this.data.rid);
+	const room = this.room = ChatRoom.findOne({_id: this.data.rid});
```

Comment:
why this change?

Reply:
This change is reverted as this change does not make any difference.